openfisca_tracker/piwik.py

```diff
@@ -1,20 +1,16 @@
 # -*- coding: utf-8 -*-
 
-import json
 from threading import Lock, Timer
 
-from unirest import post
+import grequests
 import logging
 
 log = logging.getLogger('gunicorn.error')
 BUFFER_SIZE = 30  # We send the tracked requests by group
 TIMER_INTERVAL = 3600  # We send the tracked requests every TIMER_INTERVAL seconds
 
 
-def default_callback(response):
-    return
-
-
+# Each of openfisca-core instance creates a tracker object (1 per worker).
```

Comment:
As I understand it, we have multiple gunicorn sync workers that handle/share only one instance of openfisca web api (= one flask app). This instance has one tracker. 
So, we have only one openfisca-core instance and one tracker instance.